### PR TITLE
[test][server] Minor logging change

### PR DESCRIPTION
**Pull request by majisourav99**
## [test][server] Minor logging change

Added log to indicate total server shutdown time.
## How was this PR tested?

CI
## Does this PR introduce any user-facing changes?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

File: clients/da-vinci-client/src/test/java/com/linkedin/davinci/client/StatsAvroGenericDaVinciClientTest.java [hunk 1]
```diff
@@ -102,6 +102,5 @@ public void testBatchGet() throws ExecutionException, InterruptedException {
     assertEquals(metrics.get(".test_store--multiget_success_request_key_count.Max").value(), 2.0);
     assertTrue(metrics.get(".test_store--multiget_success_request_ratio.SimpleRatioStat").value() < 1.0);
     assertTrue(metrics.get(".test_store--multiget_success_request_key_ratio.SimpleRatioStat").value() < 1.0);
-    System.out.println(metricsRepository.metrics());
```
Comment (m-nagarajan) [inline clients/da-vinci-client/src/test/java/com/linkedin/davinci/client/StatsAvroGenericDaVinciClientTest.java:105]: thanks for cleaning out these logs.

File: internal/venice-test-common/src/main/java/com/linkedin/venice/testng/VeniceSuiteListener.java [hunk 2]
```diff
@@ -1,23 +0,0 @@
-package com.linkedin.venice.testng;
```
Comment (m-nagarajan) [inline internal/venice-test-common/src/main/java/com/linkedin/venice/testng/VeniceSuiteListener.java:1]: `VeniceSuiteListener` looks to be used in `build.gradle` files.

Comment (majisourav99) [reply internal/venice-test-common/src/main/java/com/linkedin/venice/testng/VeniceSuiteListener.java:1]: good catch! i will restore it then

File: services/venice-server/src/main/java/com/linkedin/venice/server/VeniceServer.java [hunk 3]
```diff
@@ -608,6 +609,7 @@ public void shutdown() throws VeniceException {
       if (exceptions.size() > 0) {
         throw new VeniceException(exceptions.get(0));
       }
+      LOGGER.info("Shutdown completed in {} ms", LatencyUtils.getLatencyInMS(startTimeMS));
```
Comment (lluwm) [inline services/venice-server/src/main/java/com/linkedin/venice/server/VeniceServer.java:612]: If any exception is thrown before this logging line, we may not able to see it in the log? If this log is important, we can consider using the `com.linkedin.venice.utils.Timer` which extends AutoCloseable for logging.

**Review by lluwm — APPROVED** [review]
LGTM!

**Pull request merged**